test/scenarios/applicationmonitoring/read_only_csi_volume.go

```diff
@@ -0,0 +1,97 @@
+//go:build e2e
+
+package applicationmonitoring
+
+import (
+	"context"
+	"testing"
+
+	dynatracev1beta1 "github.com/Dynatrace/dynatrace-operator/src/api/v1beta1"
+	"github.com/Dynatrace/dynatrace-operator/src/kubeobjects"
+	"github.com/Dynatrace/dynatrace-operator/src/kubeobjects/address"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/components/dynakube"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/kubeobjects/deployment"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/kubeobjects/pod"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/sampleapps"
+	sample "github.com/Dynatrace/dynatrace-operator/test/helpers/sampleapps/base"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/shell"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/steps/assess"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/steps/teardown"
+	"github.com/Dynatrace/dynatrace-operator/test/helpers/tenant"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+)
+
+const (
+	csiReadOnlyVolumeEnvVar = "CSI_VOLUME_READONLY"
+	initContainerName       = "install-oneagent"
+	configPath              = "/opt/dynatrace/oneagent-paas/agent/conf"
+	ruxitAgentProcFile      = "_ruxitagentproc.conf"
```

Comment:
don't we have already consts for (some of) these somewhere in the codebase? 🤔

Reply:
Yes for the initContainerName most probably. I can check for the others. If so, should I simply export them in order to use them here?

Reply:
if its not already exported then yes I think its ok to export them to use them

Reply:
To be honest I seem only to find a constant for the default install path under the webhook package. Can you confirm, that the others are not elsewhere defined?

Reply:
https://github.com/Dynatrace/dynatrace-operator/blob/6cdc2990cd437273436b60988e0dada225efcdb4/src/config/agent_injection.go#L31

https://github.com/Dynatrace/dynatrace-operator/blob/6cdc2990cd437273436b60988e0dada225efcdb4/src/webhook/config.go#L55

https://github.com/Dynatrace/dynatrace-operator/blob/6cdc2990cd437273436b60988e0dada225efcdb4/src/webhook/mutation/pod_mutator/oneagent_mutation/config.go#L34-L35 (this is not exported)

`_ruxitagentproc.conf` is the only that is missing (so that should be fine)

Reply:
Thank you very much, I had a brain lag don't know how I could not find those :sweat_smile: